shared/getting-started/getDeviceOnDash/raspberry-pi.md

```diff
@@ -15,7 +15,7 @@ If your device still hasn't shown up on your dashboard after 10 minutes, somethi
 
 If you still can't get your device online, come on over and talk to us on our [support channel][usingSupport].
 
-__Note:__ If you have an HDMI screen attached, you should see `"Booted - Check your {{ $names.company.lower }} dashboard."` on the screen when the device boots. If instead you see rainbow colors or a black screen with a raspberry on it, it could mean that the SD card was not burned correctly or is corrupted.
+__Note:__ If you have an HDMI screen attached, you should see Balena logo on the screen when the device boots. on the screen when the device boots. If instead you see rainbow colors or a black screen with a raspberry on it, it could mean that the SD card was not burned correctly or is corrupted.
```

Comment:
"on the screen when the device boots. on the screen when the device boots."

Whoops... :wink:

Reply:
Good catch.. Can never trust a good "replace all"!